portfolio/src/main/webapp/index.html

```diff
@@ -88,5 +87,115 @@ <h1 class ="titles">Photography</h1>
         <figure class="cell cell-6"><img class="grid-pic" src="images/photos/p13.jpg"></figure>
       </secion>
     </div>
+    <div class="blog">
+      <h1 class="titles">Blog</h1>
+      <div class="container">
+        <div class="cards">
+          <a class="card" href="#">
```

Comment:
It's intentional that you want to scroll to the top of the page when this is clicked, yeah?

Reply:
It is just a dummy link until I write the blog.